conftest.py

```diff
@@ -1,22 +1,29 @@
 def pytest_addoption(parser):
     # Attaches SHARK command-line arguments to the pytest machinery.
     parser.addoption(
-        "--save_mlir",
+        "--benchmark",
         action="store_true",
         default="False",
-        help="Pass option to save input MLIR",
+        help="Pass option to benchmark and write results.csv",
     )
     parser.addoption(
-        "--save_vmfb",
```

Comment:
Don't want to leave this in ?

Reply:
There's a .vmfb that gets saved locally without the arg here if I remember correctly.
I've been meaning to get rid of / rework the deprecated pytest options for a few weeks now. I'm going to go ahead and make sure it's tracked so we can get the option back in (or at least make it easier to tell where the .vmfb is being saved otherwise).